react-hyf/react-hyf-client/src/components/Homeworkweek3.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import '../App.css';
+export class Homeworkweek3 extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+    };
+  }
+
+  fetchUsers = () => {
+    fetch(`https://cors-anywhere.herokuapp.com/https://reqres.in/api/users?page=2`)
+      .then(response => {
+        return response.json();
+      })
+      .then(res => {
+        this.setState({
+          users: res.data,
+        });
+      })
+      .catch(error => {
+        this.setState({ error, isLoading: false });
+      });
+  };
+
+  componentDidMount() {
+    this.fetchUsers();
+  }
+
+  render() {
+    const { isLoading, users, error } = this.state;
+    return (
+      <React.Fragment>
+        <h1 id="user"> Users</h1>
+        {error ? <p>{error.message}</p> : null}
```

Comment:
mostly we find this condition where we have to have a `null` value for no reason, or just to say this is a nothing value, it a correct use, but you could do the following too:

```suggestion
        {error && <p>{error.message}</p>}
```
this line now just says:
> if `error` is a truthy value the paragraph will be returned otherwise nothing will be shown, unless a zero since the zero is a falsey value and it used to be visible.